doc/api/index.rst

```diff
@@ -173,6 +173,7 @@ Input/output
 .. autosummary::
     :toctree: generated
 
+    GMTBackendEntrypoint
```

Comment:
I'm wondering if it makes more sense to have a separate category like "Xarray Integration" and put both `GMTBackendEntrypoint` and `GMTDataArrayAccessor` in the category. Also, if #3854 is implemented, `GMTDataArrayAccessor` no longer makes sense in the "Metadata" category.

Reply:
Yeah, I was wondering about this too. I can put `GMTBackendEntrypoint` in a new "Xarray Integration" section as you suggested.

Reply:
Move `GMTDataArrayAccessor` to the "Xarray Integration" section in this PR?

Reply:
> Move `GMTDataArrayAccessor` to the "Xarray Integration" section in this PR?

I think do that in #3854. Hopefully this PR can be done before that.